### PR TITLE
Update variant cooccurrence help and FAQ text

### PR DESCRIPTION
**Pull request by ksamocha**
I opened a fork and made a new branch so that I could update a few of the variant co-occurrence FAQ pages. While I have a few other changes to make, this commit is simply changing the preprint link in one of the FAQs to the publication link.

Other changes will be added as I update other FAQ/help topic pages.

### CODE REVIEW

**Review by ch-kr — APPROVED** [review]
LGTM!

**Comment by ksamocha** [comment]
This should be all of the changes that I plan to make at this point.

File: browser/help/faq/variant-cooccurrence/why-is-a-pair-of-variants-missing-from-the-downloadable-table.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+question: 'Why is a pair of variants missing from the downloadable table?'
+---
+
+We calculated inferred phasing data for all pairs of variants within a gene where both variants have a global allele frequency in gnomAD exomes <5% and are either coding, flanking intronic (from position -1 to -3 in acceptor sites, and +1 to +8 in donor sites) or in the 5’/3’ UTRs.
```
Comment (ch-kr) [inline browser/help/faq/variant-cooccurrence/why-is-a-pair-of-variants-missing-from-the-downloadable-table.md:5]:
```suggestion
We calculated inferred phasing data for all pairs of variants within a gene where both variants have a global allele frequency in gnomAD v2.1.1 exomes <5% and are either coding, flanking intronic (from position -1 to -3 in acceptor sites, and +1 to +8 in donor sites), or in the 5’/3’ UTRs.
```
just adding this work was done on v2

File: browser/help/topics/variant-cooccurrence.md [hunk 2]
```diff
@@ -3,8 +3,8 @@ id: variant-cooccurrence
...
+To help with these challenges we have released inferred phasing data for all pairs of variants within a gene where both variants have a global allele frequency in gnomAD exomes <5% and are either coding, flanking intronic (from position -1 to -3 in acceptor sites, and +1 to +8 in donor sites) or in the 5’/3’ UTRs. On the browser, over 5.3 billion pairs of variants across 19,685 genes can be queried. We have also released a file of 20,921,100 pairs of variants that were seen at least once in the same individual. As a note: if a pair of qualifying variants is not in this file, it indicates that the two variants were never seen together in the same individual, and therefore we would infer that these variants are in trans. To learn more about this approach please read our [blog post](https://gnomad.broadinstitute.org/news/2021-07-variant-co-occurrence-phasing-information-in-gnomad/) or [paper](https://www.nature.com/articles/s41588-023-01608-3).
```
Comment (ch-kr) [inline browser/help/topics/variant-cooccurrence.md:8]:
```suggestion
To help with these challenges, we have released inferred phasing data for all pairs of variants within a gene where both variants have a global allele frequency in gnomAD v2.1.1 exomes <5% and are either coding, flanking intronic (from position -1 to -3 in acceptor sites, and +1 to +8 in donor sites), or in the 5’/3’ UTRs. On the browser, over 5.3 billion pairs of variants across 19,685 genes can be queried. We have also released a file of 20,921,100 pairs of variants that were seen at least once in the same individual. As a note: if a pair of qualifying variants is not in this file, it indicates that the two variants were never seen together in the same individual, and therefore we would infer that these variants are in trans. To learn more about this approach please read our [blog post](https://gnomad.broadinstitute.org/news/2021-07-variant-co-occurrence-phasing-information-in-gnomad/) or [paper](https://www.nature.com/articles/s41588-023-01608-3).
```

File: browser/help/topics/variant-cooccurrence.md [hunk 3]
```diff
@@ -3,8 +3,8 @@ id: variant-cooccurrence
...
+Additionally, we have incorporated cumulative counts of gnomAD individuals carrying pairs of rare co-occurring variants within genes in the gnomAD v2 browser, across various allele frequencies and functional consequences. These counts can be used to evaluate how frequently rare variant co-occurrence is observed in a large reference population. To read more about this approach, how to access this data in the browser, as well as limitations please see our [blog post](https://gnomad.broadinstitute.org/news/2023-03-variant-co-occurrence-counts-by-gene-in-gnomad/). Additionally you can read our paper [here](https://www.nature.com/articles/s41588-023-01608-3).
```
Comment (ch-kr) [inline browser/help/topics/variant-cooccurrence.md:10]:
```suggestion
Additionally, we have incorporated cumulative counts of gnomAD individuals carrying pairs of rare co-occurring variants within genes in the gnomAD v2 browser across various allele frequencies and functional consequences. These counts can be used to evaluate how frequently rare variant co-occurrence is observed in a large reference population. To read more about this approach, how to access this data in the browser, as well as limitations please see our [blog post](https://gnomad.broadinstitute.org/news/2023-03-variant-co-occurrence-counts-by-gene-in-gnomad/). Additionally you can read our paper [here](https://www.nature.com/articles/s41588-023-01608-3).
```

**Review by ch-kr — APPROVED** [review]
a few nitpicks, otherwise LGTM

**Comment by rileyhgrant** [comment]
Thanks for doing this Kaitlin and for the review KC :)

I just added a smol commit to update a snapshot to appease the CI, and since it seems like activity is complete on the changes front will merge this into `main`.

**Pull request merged**